src/Processor/AdvancedActions/AnonymizeCustomersNotLoggedBeforeProcessor.php

```diff
@@ -22,11 +23,15 @@ class AnonymizeCustomersNotLoggedBeforeProcessor implements AdvancedActionsFormD
     /** @var ParameterBagInterface */
     private $parameterBag;
 
-    public function __construct(EntityManagerInterface $entityManager, AnonymizerProcessor $anonymizerProcessor, ParameterBagInterface $parameterBag)
+    /** @var FlashBagInterface */
+    private $flashBag;
```

Comment:
You can type hint that

Reply:
No i can't, else the compatibility with php7.3 won't work anymore